### PR TITLE
[In progress] Updating kinematic: ini files

### PR DESCRIPTION
**Pull request by rsantos88**
Updating kinematic: ini files

### CODE REVIEW

**Comment by jgvictores** [comment]
@rsantos88 How about removing old/unused/useless?

**Comment by jgvictores** [comment]
> @rsantos88 How about removing old/unused/useless?

PS: Yes, this was proposed at https://github.com/roboticslab-uc3m/teo-configuration-files/issues/7

File: share/kinematics/leftArmKinematics-fetch.ini [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+numLinks 8
+
+link_0 (offset   0.0) (D  0.0    ) (A  0.0     ) (alpha -90.0) (mass 0       ) (cog 0 0 0) (inertia 0 0 0)
+link_1 (offset -90.0) (D  0.0    ) (A  0.0     ) (alpha -90.0) (mass 0       ) (cog 0 0 0) (inertia 0 0 0)
+link_2 (offset -90.0) (D -0.32901) (A  0.0     ) (alpha -90.0) (mass 1.750625) (cog 0 0 0) (inertia 0 0 0)
+link_3 (offset   0.0) (D  0.0    ) (A  0.0     ) (alpha  90.0) (mass 0       ) (cog 0 0 0) (inertia 0 0 0)
+link_4 (offset   0.0) (D -0.202  ) (A  0.0     ) (alpha -90.0) (mass 2.396   ) (cog 0 0 0) (inertia 0 0 0)
+link_5 (offset -90.0) (D  0.0    ) (A -0.09    ) (alpha   0.0) (mass 0.300   ) (cog 0 0 0) (inertia 0 0 0)
+link_6 (offset -90.0) (D  0.0    ) (A  0.0     ) (alpha  90.0)
```
Comment (jgvictores) [inline share/kinematics/leftArmKinematics-fetch.ini:15]: `link_5` and `link_6` redundant with `HN`

File: share/kinematics/rightArmKinematics-fetch.ini [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+numLinks 8
+
+link_0 (offset   0.0) (D  0.0    ) (A  0.0      ) (alpha -90.0) (mass 0       ) (cog 0 0 0) (inertia 0 0 0)
+link_1 (offset -90.0) (D  0.0    ) (A  0.0      ) (alpha -90.0) (mass 0       ) (cog 0 0 0) (inertia 0 0 0)
+link_2 (offset -90.0) (D -0.32901) (A  0.0      ) (alpha -90.0) (mass 1.750625) (cog 0 0 0) (inertia 0 0 0)
+link_3 (offset   0.0) (D  0.0    ) (A  0.0      ) (alpha  90.0) (mass 0       ) (cog 0 0 0) (inertia 0 0 0)
+link_4 (offset   0.0) (D -0.202  ) (A  0.0      ) (alpha -90.0) (mass 2.396   ) (cog 0 0 0) (inertia 0 0 0)
+link_5 (offset -90.0) (D  0.0    ) (A -0.09     ) (alpha   0.0) (mass 0.300   ) (cog 0 0 0) (inertia 0 0 0)
+link_6 (offset -90.0) (D  0.0    ) (A  0.0      ) (alpha  90.0)
```
Comment (jgvictores) [inline share/kinematics/rightArmKinematics-fetch.ini:15]: `link_5` and `link_6` redundant with `HN`

**Review by jgvictores — CHANGES_REQUESTED** [review]

**Comment by rsantos88** [comment]
It's already done but, I'll close this PR and reopen a new PR with less changes

**Comment by jgvictores** [comment]
Closed as `invalid` in favor of https://github.com/roboticslab-uc3m/teo-configuration-files/pull/21

**Pull request closed without merging**